### PR TITLE
Add rivtd api

### PR DESCRIPTION
**Pull request by JHowix**
Adding RIVTD Track 2 API

### CODE REVIEW

File: apis/match-to-id/api/riv-demo-match-to-id.yaml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        - ImageData
+
+    Template:
+      type: object
+      description: |
+        Data transfer object for a template.
+      properties:
+        Template:
+          description: The template data, encoded as a base64 string.
+          type: string
```
Comment (rbell13) [inline apis/match-to-id/api/riv-demo-match-to-id.yaml:43]: please add the line after this:
format: byte

File: apis/match-to-id/api/riv-demo-match-to-id.yaml [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+          description: |
+            The email address of an engineer or other technical resource to contact in the event of
+            an error running your service.
+          example: john@mdtf.org
+        RecommendedCPUs:
+          type: number
+          format: float
+          description: The recommended allocation of CPUs for the deployed docker container.
+          example: .5
+          maximum: 8
```
Comment (rbell13) [inline apis/match-to-id/api/riv-demo-match-to-id.yaml:110]: are these maximums correct?

Comment (JHowix) [reply apis/match-to-id/api/riv-demo-match-to-id.yaml:112]: yes these are OK according to Bharani

File: apis/match-to-id/api/riv-demo-match-to-id.yaml [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+  schemas:
+
+    Image:
+      type: object
+      description: |
+        Data transfer object for an image.
+      properties:
+        ImageData:
+          type: string
```
[the remark this answers is not in the document]
Comment (JHowix) [reply apis/match-to-id/api/riv-demo-match-to-id.yaml:29]: done

File: apis/match-to-id/api/riv-demo-match-to-id.yaml (hunk 1 above)
Comment (JHowix) [reply apis/match-to-id/api/riv-demo-match-to-id.yaml:44]: done

**Review by rbell13 — APPROVED** [review]

**Pull request merged**